repair/row_level.cc

```diff
@@ -1669,6 +1676,7 @@ class repair_meta {
         // retransmission at messaging_service level, so no message will be retransmited.
         rlogger.trace("Calling get_combined_row_hash_handler");
         return with_gate(_gate, [this, common_sync_boundary = std::move(common_sync_boundary)] () mutable {
+            _cf.update_off_strategy_trigger();
```

Comment:
why are we updating trigger unconditionally here? and below...

Reply:
Because there is repair traffic. It is fine to update the timer whenever there is repair traffic.